src/ansys/geometry/core/designer/component.py

```diff
@@ -895,6 +898,46 @@ def create_surface_from_face(self, name: str, face: Face) -> Body:
         self._master_component.part.bodies.append(tb)
         return Body(response.id, response.name, self, tb)
 
+    @protect_grpc
+    @check_input_types
+    @ensure_design_is_active
+    @min_backend_version(24, 2, 0)
```

Comment:
Are we sure this is available in 24R2 already?

Reply:
It is not, changing this